.github/workflows/publish_images_on_schedule.yml

```
@@ -0,0 +1,28 @@
+name: Schedule Build Docker Images For Main Images And Nightly
```

```suggestion
name: Cron job for building and publishing Docker images
```

I think the file name should include docker. How does `publish_docker_images_cron.yml` sound?

---

ping

---

You changed the name field, but you missed my main point which is having a consistent casing scheme.

I would want to see all lowercase except for the first letters of the entry and proper nouns

The current entry:
` name: Schedule Build Docker Images For Main Images And Nightly `

---

Fixed. I missed this one since it is hidden with other issues. Sorry for missing it.

> sing commit or pull request references in commit messages. It can lead to so

I will be more careful about that Thanks for the warning